enterprise/tpa_pipeline.py

```diff
@@ -67,3 +72,24 @@ def handle_enterprise_logistration(backend, user, **kwargs):
         user_id=user.id
     )
     enterprise_customer_user.update_session(request)
+
+
+def get_user_from_social_auth(tpa_provider, tpa_username):
+    """
+    Find the LMS user from the UserSocialAuth model by the provide `tpa_username`.
```

Comment:
Nit: Update comment to correctly reflect the arguments, if you are going to mention them.

Reply:
Done